### ISSUE TITLE
release `1.0.1-a.22` for `mac intel`:  download page return 404, due to wrong href in zen downloads page

### ISSUE THREAD

**Issue by tresorama** [zen-browser/desktop#3272]
## TLDR: link on downloads page of zen website is incorrect for Mac intel, resulting in 404.

I'm on Mac Intel, on 1.0.1-a.21.

Zen Browser showed me a popup (top right corner) to update manually by downloading the new release.

On click it redirects me to https://zen-browser.app/download , then i select Mac > Intel and the link redirect me to 
https://github.com/zen-browser/desktop/releases/download/1.0.1-a.22/zen.macos-x86_64.dmg
I see the 404 of github.

II checked with the release page of Githib repo, and the correct link should be
https://github.com/zen-browser/desktop/releases/download/1.0.1-a.22/zen.macos-x64.dmg

Note the correct link hasn't `x86_` in file name

### Version

1.0.1-a.21

### What platform are you seeing the problem on?

macOS - Intel

**Comment by tresorama** [comment]
Checked now. The link is now correct. 
Closing this

**Comment by bartimaeus** [comment]
@tresorama this appears to be broken for the latest build too.

On click it redirects me to [https://zen-browser.app/download](https://zen-browser.app/download), then i select Mac > Intel and the link redirect me to
[https://github.com/zen-browser/desktop/releases/download/1.0.2-b.0/macos-x86_64.dmg](https://github.com/zen-browser/desktop/releases/download/1.0.2-b.0/macos-x86_64.dmg)
I see the 404 of github.

I think the link should be
```diff
-https://github.com/zen-browser/desktop/releases/download/1.0.2-b.0/macos-x86_64.dmg
+https://github.com/zen-browser/desktop/releases/download/1.0.2-b.0/zen.macos-x86_64.dmg
```

**Comment by tresorama** [comment]
Reopen because the [problem is present again](https://github.com/zen-browser/desktop/issues/3272#issuecomment-2532751258) when downloading `1.0.2-b.0`

**Comment by mr-cheffy** [comment]
Should be fixed now 👍🏽
